resources/event-bus/values.yaml

```diff
@@ -30,8 +30,12 @@ global:
     maxEventTypeLength: 253
     maxEventTypeVersionLength: 4
     resources:
+      requests:
```

Comment:
Setting `requests` will automatically change the QoS class from _Guaranteed_ to _Burstable_. Are we sure this is desired?

Reply:
it's a Work In Progress